reducer/src/test/java/com/graphicsfuzz/reducer/ReductionDriverTest.java

```diff
@@ -542,7 +542,7 @@ public void testReductionWithUniformBindings() throws Exception {
     ShaderJob shaderJob = new GlslShaderJob(Optional.empty(),
         pipelineInfo,
         fragShader);
-    shaderJob.makeUniformBindings();
+    shaderJob.makeUniformBindings(Optional.empty());
```

Comment:
We should add a reducer test that demonstrates that push constants get preserved during reduction.

Reply:
Done, and revealed a few unrelated bugs.